EssentialsDiscord/src/main/java/net/essentialsx/api/v2/events/discord/DiscordRelayEvent.java

```diff
@@ -89,6 +93,15 @@ public void setFormattedMessage(final String formattedMessage) {
         this.formattedMessage = formattedMessage;
     }
 
+    /**
+     * Gets the users that will see this relayed message in their chat.
+     * This list is mutable, removing a player from it will hide the message from them.
+     * @return The mutable list of users.
+     */
```

Comment:
```suggestion
    /**
     * Gets the users that will be sent the relayed message.
     * The returned list is mutable. Removing a player from it will hide the message from them.
     * @return The mutable list of users.
     */
```